src/stream/src/common/log_store_impl/kv_log_store/reader.rs

```diff
@@ -192,6 +192,10 @@ pub struct KvLogStoreReader<S: StateStoreRead> {
     rewind_delay: RewindDelay,
 
     align_epoch_on_init: bool,
+
```

Comment:
It seems we don't have to store the `rewind_start_offset` as a field. We always use the newly passed `rewind_start_offset` and never really use the field.